registry/remote/example_test.go

```diff
@@ -79,6 +92,13 @@ func TestMain(m *testing.M) {
 			w.WriteHeader(http.StatusCreated)
 		case p == fmt.Sprintf("/v2/%s/manifests/%s", exampleRepositoryName, exampleTag) && m == "PUT":
 			w.WriteHeader(http.StatusCreated)
+		case p == fmt.Sprintf("/v2/%s/manifests/%s", exampleRepositoryName, exampleReferenceManifestDescriptor.Digest) && m == "GET":
+			w.Header().Set("Content-Type", ocispec.MediaTypeImageManifest)
```

Comment:
I think the Content-Type here should be `artifactspec.MediaTypeArtifactManifest` instead of `ocispec.MediaTypeImageManifest`, but the Fetch() method requires this content type.

Reply:
Because the manifest is declared with wrong type. See comments around var statements.